cmd/ctr/run_windows.go

```diff
@@ -86,7 +86,7 @@ func setHostNetworking() containerd.SpecOpts {
 
 func newContainer(ctx gocontext.Context, client *containerd.Client, context *cli.Context) (containerd.Container, error) {
 	var (
-		// ref          = context.Args().First()
+		// imageName          = context.Args().First()
```

Comment:
Just remove this line?

Reply:
I'll be adding this line back in my PR to add Windows snapshotter/image stuff in a few days. Probably better to keep it so that we can ensure the names are in sync